src/fragments/lib/auth/flutter/signin_web_ui/20_platform_specific_setup.mdx

```diff
@@ -92,3 +68,50 @@ Add the following entry to the URL scheme in the `Info.plist` file in your app's
      <!-- ... -->
      </dict>
 ```
+</Block>
+
+<Block name="Developer Preview (Mobile, Web & Desktop)">
+
+## Web Platform Setup
+
+To use Hosted UI in your Flutter web application locally, you must run the app with the `--web-port=3000` argument (with the value being whichever port you assigned to localhost host when configuring your redirect URIs).
+
+## Android Platform Setup
+
+Add the following `queries` element to the `AndroidManifest.xml` file in your app's `android/app/src/main` directory, as well as the following `intent-filter` to the `MainActivity` in the same file.
+
+Replace `myapp` with your redirect URI scheme as necessary:
+
+```xml
+<queries>
+    <intent>
+        <action android:name=
+            "android.support.customtabs.action.CustomTabsService" />
+    </intent>
+</queries>
+<application>
+  ...
+  <activity
+        android:name=".MainActivity">
+        <intent-filter>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+            <data android:scheme="myapp" />
+        </intent-filter>
+  </activity>
+  ...
+</application>
+```
+
+## macOS
+
+If you haven't already, open XCode and enable the App Sandbox capability and then select "Outgoing Connections (Client)" under "Network".
```

Comment:
Incoming is the hosted UI specific one that's needed. Outgoing should be configured for all projects.

Reply:
Point to the pics you have from Xcode here.